packages/custom-elements-manifest-tools/src/lib/reference-string.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Module, Reference} from 'custom-elements-manifest/schema';
+
+/**
+ * Serializes a reference to a string, as used in the GraphQL API of the
+ * custom elements catalog.
+ *
+ * TODO (justinfagnani): define format for package and module relative refs.
+ * Take a Reference object instead of individual params.
+ */
+export const referenceString = (
+  packageName: string,
+  mod: Module,
+  name: string
+) => {
+  return `${packageName}/${mod.path}#${name}`;
```

Comment:
Can `mod.path` be an empty string for the main module? If so, is `pkg/#name` ok here?

Reply:
Right now it can't be, because we don't have the information to map a bare specifier to a module. We'll need the package.json for that. So currently it throws an error.